raiden_libs/network/matrix/client.py

```diff
@@ -304,13 +308,7 @@ def call(self, callback, *args, **kwargs):
 
     def _sync(self, timeout_ms=30000):
         """ Copy-pasta from MatrixClient, but add 'account_data' support to /sync """
-        try:
```

Comment:
As far as I can see we still need to swallow the exceptions that happen when `sync_thread.kill()` is called (see comment in `stop_listener_thread`). Why is this `try` block removed?

Reply:
@jomuel Now, it's being handled in the `listen_forever` loop above. The exception still is swallowed, with only a warning.